packages/components/src/components/NewModal/NewModal.tsx

```diff
@@ -62,6 +53,13 @@ const HeadingContainer = styled.div`
     display: flex;
     flex-direction: column;
     flex: 1;
+    overflow: hidden;
+`;
+
+const Heading = styled(H3)`
```

Comment:
I'm not happy about this 🤔

Shouldn't we create some ellipsis prop directly in Heading? (Not necessarilu in this PR)

Reply:
Yeah, me neither. Due to how ellipsis work in css it can't be done any other way than directly styling the H3. I tried the Truncate component but it didn't work.